config/webpack.cozyui.config.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const ExtractTextPlugin = require('extract-text-webpack-plugin')
+
+const build = process.env.NODE_ENV === 'production'
+
+module.exports = {
+  resolve: {
+    extensions: ['.styl']
+  },
+  module: {
+    loaders: [
+      {
+        test: /\.styl$/,
+        loader: ExtractTextPlugin.extract([
+          'css-loader?importLoaders=1',
+          'postcss-loader',
+          'stylus-loader'
+        ])
+      }
+    ]
+  },
+  plugins: [
+    new ExtractTextPlugin(build ? 'app.[hash].css' : 'app.css')
+  ],
+  stylus: {
+    use: [require('cozy-ui/lib/stylus')()]
+  }
+}
```

Comment:
I think we should also have `config/webpack.cozyclient.js` for cozy-client-js in the same spririt